docs/motivation-and-use-cases.md

```diff
@@ -0,0 +1,138 @@
+# IPFS cluster use cases 
+
+These are some WIP ipfs-cluster use case sketches.  These are not formal use cases and are more accurately groups of related use cases; they could be further decomposed into the more narrowly scoped operations found in formal use cases.
+
+## ipfs node mirror
+Actors:
+	ipfs user
+
+Description: An ipfs user with multiple machines wants to run their ipfs node with better replication or availability guarantees.  The user creates an ipfs cluster across machines.  Adding content to ipfs automatically triggers a pin in the cluster according to some predetermined replication strategy.  The user advertises multiaddresses from all the ipfs daemons as multiaddresses of the ipfs node mirror.
+
+Thoughts: This use case, like others, requires some mechanism for automatic pinning upon adding to ipfs.  Depending on the adding mechanism this might be straightforward functionality to add to cluster, for example cluster's ipfs proxy add endpoint will probably eventually do this by default.  However in some cases, such as if each machine writes to ipfs over the fuse interface, this would be more difficult.  A mirrored node could potentially advertise itself as an ipfs node and advertise the cluster ipfs proxy endpoint addresses as its multiaddresses.
```

Comment:
Following up on an old thread -- do we do this now? Is this how Cluster works? -->

An IPFS user with multiple computers wants to make sure their content is always available on the IPFS network. That means they need to make sure their computer is always on and connected to the network, or that they’ve made copies on other computers so that if one goes down, there’s still copies of their content via other hard drives on the network.

Before IPFS Cluster, this user would have needed to do [some really annoying thing]. Now, this user can use Cluster to create a connected set of locations to store their content. When the user adds content to IPFS, Cluster automatically creates copies across the user’s computers according to their settings. Perhaps this user is very worried about losing access to this content, and so sets Cluster to make 100 copies; perhaps the user is less worried about consistent access, and so sets Cluster to make only one other copy.

As computers come online and offline, Cluster also takes care of maintaining the user’s replication strategy: the user might always want 100 copies of their content available, but the exact computers storing this information might change. There will, however, always be 100 copies available.

**Is this right?** (I'm hoping to slowly get a handle on how to describe what Cluster does so we can make a more plain-language version of our intro/high-level docs.)